app/controllers/idv/please_call_controller.rb

```diff
@@ -1,14 +1,25 @@
 module Idv
   class PleaseCallController < ApplicationController
+    include FraudReviewConcern
+
     before_action :confirm_two_factor_authenticated
+    before_action :handle_fraud_rejection
+    before_action :confirm_fraud_pending
 
     FRAUD_REVIEW_CONTACT_WITHIN_DAYS = 14.days
 
     def show
       analytics.idv_please_call_visited
-      pending_at = current_user.fraud_review_pending_profile.fraud_review_pending_at ||
-                   Time.zone.today
+      pending_at = current_user.fraud_review_pending_profile.fraud_review_pending_at
       @call_by_date = pending_at + FRAUD_REVIEW_CONTACT_WITHIN_DAYS
     end
+
+    private
+
+    def confirm_fraud_pending
+      if !fraud_review_pending?
+        redirect_to account_url
```

Comment:
Should we check if they're in the midst of IdV and redirect to idv_url? Or redirect there and let it check as needed?

Reply:
Do we have a reliable pattern for checking if a user is in the midst of IdV somewhere that I could use? I'm not sure this is the right place to introduce that pattern if not. A user should not be sent here if they are in the midst of IdV (and they aren't being sent here evidenced by the lack of 500s errors observed from this controller).

Reply:
As far as I can tell, only IdV controllers redirect to this controller with idv_please_call_url. There's now a user_please_call_url for suspended users that is used from application controller.

And, I looked at idv controller and it doesn't have a before action to redirect to account_url, so this seems fine. Good point about not seeing 500s.